metrics/README.md

```diff
@@ -1,10 +1,13 @@
 # Bigquery metrics
```

Comment:
Personally rather reading this Tolkienesque snippet (that I'm unlikely to read anyway unless as a last resort) I would prefer a couple concrete examples and an explanation for each piece